start/scaling-up/scaling-up.md

```diff
@@ -0,0 +1,70 @@
+[title]: - "Scale Up your Workflow on OSG Connect"
+[TOC]
+
+## Background
+
+Much of OGS's computing power comes from the ability to run a large number 
+of jobs simulateously. Many HTC-friendly workflows involve running the same 
+job multiple times with no or minor changes between them. Here, we will 
+talk about how to submit multiple jobs at once using the `queue` command as 
+well as some important considerations and tips for scaling up your HTCondor jobs. 
+For a hands-on example of these concepts, checkout the [OSG Connect Quickstart]
+(https://support.opensciencegrid.org/support/solutions/articles/5000633410-osg-connect-quickstart). 
+
+## Things to Consider
+
+In order to avoid confusion, let's review some HTCondor terminiology:
+ - When you queue a single submit script using the `condor_submit` command, this is 
+called a **cluster** and can consist of one more individual jobs.
+ - Each individual job within a cluster is called a **process**. Therefore, each 
+*cluster* can be made up of one or more *processees*. Each proceess lands on its 
+own job slot and is handled individually from the other processes in the cluster.
+
+### Handling Output
+
+HTCondor has built in features that make submitting multiple jobs with the same 
+submit script possible and easy. However, this means that you will need to ensure 
+that the output created by each individual process will be uniquely identified so 
+it is not overwritten by the output from other processes.
+
+In order to uniquely identify each job output, you can use cluster and process IDs
+in the submit script. For example, to route each job log into it's own file, the 
+following line would ensure that each job's log file would end up with a unique file 
+name:
+
+	log = job.$(Cluster).$(Process).log
+
+For a working example on how to use this in a submit file, and additional applications,
+please see [Job 3 of the OSG Connect Quickstart guide]
+(https://support.opensciencegrid.org/support/solutions/articles/5000633410-osg-connect-quickstart#job-3-submitting-jobs-concurrently)

```

Comment:
It would also be nice to refer to examples where users provide a list of values or use a glob instead of using the `$(Process)` numbers. If we don't show in detail anywhere in our guides (yet), maybe refer to HTCondor Week talks?

Reply:
Added a section on Submitting Multiple Jobs with a Single Script which mentions this option and links to CHTC's doc page on it.